index.html

```diff
@@ -22,6 +22,7 @@
   </header>
```

Comment:
- [ ] **[OPTIONAL] Good Job implementing the changes for the contact form :clap: ,However please make sure the hamburger icon is gone in the desktop version and the navbar is similar to what it is in figma.** 

**Your current Navbar design :point_down:** 
![image](https://user-images.githubusercontent.com/97521557/221359927-9a48e285-5f4d-44c8-8aaa-aa96894c817d.png)

**Figma Navbar design :point_down:** 
![image](https://user-images.githubusercontent.com/97521557/221360129-f2d1b716-3bec-46fc-b1b6-e3f8c78b53ca.png)